index.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Damon Romano, Quiz: HTML/CSS</title>
+    <link rel="stylesheet" type="text/css" href="quiz.css">
+  </head>
+
+  <body>
+    <!-- Each card should be an article.-->
+    <!-- -------------------------- FIRST PRODUCT CARD ---------------------------->
```

Comment:
One rule to keep in mind for HTML comments: “--” is not permitted inside a comment, but e.g. “- -” is.